### DISCUSSION TITLE
Feature Request: Jump to type definition inside popup

### DISCUSSION THREAD

**Discussion by Zomatree** [microsoft/pylance-release#2636] [Enhancement]
Similar to how [rust-analyzer](https://github.com/rust-analyzer/rust-analyzer) has the ability to jump to the defenition of types used in a line, it would be nice if pylance also has this ability
![image](https://user-images.githubusercontent.com/39768508/151756963-b310feed-9584-4d90-ba15-d696911699a9.png)

![image](https://user-images.githubusercontent.com/39768508/151756713-a585e682-c92c-4fbe-ba61-3ff462d05010.png)
This allows you to jump to where each of the types are defined inside your code and your dependence's code

**Comment by judej** [comment]
Thanks for the suggestion. Will add it to the enhancement list.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
